### PR TITLE
Fixing bad examples

### PR DESCRIPTION
**Pull request by SteveMutungi254**
Correcting examples based on PSModuleAnalyzer results.

### CODE REVIEW

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Enable-EntraAzureADAlias.md [hunk 1]
```diff
@@ -9,25 +19,28 @@ schema: 2.0.0
...
 
-```
+```powershell
 Enable-EntraAzureADAlias
 ```
 
 ## Description
-{{ Fill in the Description }}
+
+Enables aliases for Azure AD modules.
```
Comment (alexandair) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Enable-EntraAzureADAlias.md:33]: Enables aliases for Azure AD commands.

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Enable-EntraAzureADAlias.md [hunk 2]
```diff
@@ -9,25 +19,28 @@ schema: 2.0.0
...
-### Example 1
+### Example 1: Enable aliasing
+
 ```powershell
-PS C:\> {{ Add example code here }}
+Enable-EntraAzureADAlias
 ```
 
-{{ Add example description here }}
+Enables all Azure AD prefixes for the current PowerShell session.
```
Comment (alexandair) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Enable-EntraAzureADAlias.md:43]: Enables aliases with AzureAD prefix for the current PowerShell session.

**Review by alexandair — CHANGES_REQUESTED** [review]

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Enable-EntraAzureADAlias.md (hunk 2 above)
Comment (SteveMutungi254) [reply module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Enable-EntraAzureADAlias.md:43]: Good catch. Applied the recommendation.

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Enable-EntraAzureADAlias.md (hunk 1 above)
Comment (SteveMutungi254) [reply module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Enable-EntraAzureADAlias.md]: Good catch. Applied the recommendation.

**Comment by SteveMutungi254** [comment]
Applied changes based on recommendations from [PSModuleAnalyzer](https://github.com/HCRitter/PSModuleAnalyzer) by @HCRitter

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**